src/rules/prefer-expect-assertions.ts

```diff
@@ -52,10 +52,10 @@ type MessageIds =
   | 'suggestAddingAssertions'
   | 'suggestRemovingExtraArguments';
 
-const suggestions: Array<[MessageIds, string]> = [
-  ['suggestAddingHasAssertions', 'expect.hasAssertions();'],
-  ['suggestAddingAssertions', 'expect.assertions();'],
-];
+// const suggestions: Array<[MessageIds, string]> = [
+//   ['suggestAddingHasAssertions', 'expect.hasAssertions();'],
+//   ['suggestAddingAssertions', 'expect.assertions();'],
+// ];
```

Comment:
why is this commented out instead of deleted?

Reply:
Oh whoops, cause I forgot about it 😅 will clean it up tomorrow along with a few other comments we've got lying around